src/wallet/rpcdump.cpp

```diff
@@ -510,3 +534,105 @@ UniValue dumpwallet(const UniValue& params, bool fHelp)
     file.close();
     return NullUniValue;
 }
+
+
+UniValue importmulti(const UniValue& params, bool fHelp)
+{
+    if (fHelp || params.size() < 1 || params.size() > 2)
+        throw runtime_error(
+            "importmulti '[{\"type\":\"privkey\",\"value\":\"mkjjX...\"},...]' (rescan) \n\n"
```

Comment:
What about the second argument be a JSON object? Here it would be `{ "rescan": true }`. But it could have more options, for instance, one that would rejects addresses with timestamps belonging to pruned blocks.